cmake/Modules/winconf.cmake

```diff
@@ -2,31 +2,45 @@
 
 list(APPEND CMAKE_LIBRARY_PATH "$ENV{LIB}")
 
-# We want to link with static version of
-# libtorrent
-set(LibtorrentRasterbar_USE_STATIC_LIBS True)
 set(LibtorrentRasterbar_CUSTOM_DEFINITIONS
+    -DBOOST_ASIO_DISABLE_CONNECTEX
     -DBOOST_EXCEPTION_DISABLE
-    -DBOOST_SYSTEM_STATIC_LINK=1
     -DTORRENT_USE_OPENSSL
-    -D__USE_W32_SOCKETS
-    -D_FILE_OFFSET_BITS=64)
+    -DTORRENT_DISABLE_RESOLVE_COUNTRIES)
+
+set(LibtorrentRasterbar_CUSTOM_BOOST_DEPENDENCIES system)
+
+# If we want to link with static version of libtorrent
+#set(LibtorrentRasterbar_USE_STATIC_LIBS True)
+#list(APPEND LibtorrentRasterbar_CUSTOM_DEFINITIONS
+#    -DBOOST_SYSTEM_STATIC_LINK=1)
+
+# and boost
+#set(Boost_USE_STATIC_LIBS  True)
+#set(Boost_USE_STATIC_RUNTIME True)
 
 add_definitions(-DUNICODE
     -D_UNICODE
     -DWIN32
     -D_WIN32
     -DWIN32_LEAN_AND_MEAN
-    -DNTDDI_VERSION=0x05010000
-    -D_WIN32_WINNT=0x0501
-    -D_WIN32_IE=0x0501
+    -DNTDDI_VERSION=0x06010000
```

Comment:
you should also update conf.pri.windows. perhaps in another commit.

Reply:
Ok.
All other comments are addressed.

Reply:
I mean you should/can include that commit in this PR.

Reply:
It's about CMake. So I'll do it in another one.

Reply:
At least I would revert it in this PR. It is better to have a single commit just for it.

Reply:
Well, let it be so.
Reverted.